### DISCUSSION TITLE
How do you Login

### DISCUSSION THREAD

**Discussion by daniel-kayode** [idurar/idurar-erp-crm#1141] [Q&A]
I have containerize the Frontend, Backend and Mangodb running on my Homelab, how do I login into the app. I have searched the whole repo and am still unable to login.  Please how can I login? or How can I register my email and password? 
 
![image](https://github.com/idurar/idurar-erp-crm/assets/71196354/02964125-2ab8-49ea-b741-de5522ce5448)
![image](https://github.com/idurar/idurar-erp-crm/assets/71196354/48f08f47-20b7-42d0-a75b-fe43f6a8b9a6)

**Comment by AhsanSarwar0413** [comment]
@salahlalami I also wanted to test the signup flow which causing an issue but I can't have access to it in local environment plus I am also facing login issue same as stated above.
